src/add-remove-edit.js

```diff
@@ -0,0 +1,31 @@
+import * as ls from './local-storage.js';
+
+export const addItem = (input, items) => {
+  items.push({
+    description: input.value,
+    complete: false,
+    index: items.length,
```

Comment:
Let's use a unique ID for each object being pushed into the array so that when we want to delete an object, it doesn't delete all objects with the same length because they have the same ID number 🙂 
 You could use this [ID generator](https://www.npmjs.com/package/uuid) for unique ID 👍

Reply:
On the one hand, I have been very careful every time, making sure duplicate ids DONT happen. In fact, as my debugging has repeatedly shown, my code crashes and burns if there are repeat ids. 
Also, they asked for index values as numbers by which the list is arranged.

On the other hand, unique ids are a great idea, thank you. I'll try to see if something can be done...